release-calendar/src/client/components/ChannelTable.tsx

```diff
@@ -16,19 +16,45 @@
 
 import '../stylesheets/channelTable.scss';
 import * as React from 'react';
+import {ApiService} from '../api-service';
 import {Channel} from '../../types';
+import {getCurrentReleases} from '../models/promotion-current';
```

Comment:
nit: rename `promotion-current` to something like `current-releases` to stay consistent